packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -202,6 +202,10 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * Theme provided by styled() function.
    */
   theme?: ITheme;
+  /**
+   * Optional read only state applied to tag picker. Removes the clear button from the TagItem pill.
+   */
+  readOnly?: boolean;
```

Comment:
Not sure we should add this prop on the `IBasePickerProps` which is supposed to be generic and it only pipes to a Tag picker. @joschect where would be the best place to add this prop so it only pipes for `TagPicker` scenario?

Reply:
I don't actually think we can add readonly as is. I'm pretty sure you'd still be able to delete each item with just backspace. Is readonly just supposed to remove the X? That should be easily achieved with a different Tag item and different styles.

Reply:
Per @joschect and my conversation I'll be removing the readOnly ability and add that functionality in a later PR